tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -292,10 +363,46 @@ void ordering_test_cases() {
         std::stack<SynthOrdered> b{std::deque<SynthOrdered>{10, 20, 40}};
         ordered_containers_test(a, a, b);
     }
-}
+    { // sub_match
+        const std::string s1{"cats"};
+        const std::string s2{"meow"};
+        const std::regex all(".*");
+        const std::regex each(".");
+        std::smatch m1, m2, m3, m4;

```

Comment:
We conventionally avoid defining multiple variables on a single line, although it is sure convenient here.

Reply:
Should I change it? It is pretty convenient here.

Reply:
I think it's worth changing because following consistent global precedent makes it easier for contributors to look at pre-existing code and imitate its style. It's only 3 additional lines, not 20.